.gitignore

```diff
@@ -1,6 +1,10 @@
 # Created by https://www.toptal.com/developers/gitignore/api/visualstudio,reactnative,react,git,ruby,yarn,node,xcode,android,intellij,fastlane
 # Edit at https://www.toptal.com/developers/gitignore?templates=visualstudio,reactnative,react,git,ruby,yarn,node,xcode,android,intellij,fastlane
 
+# OSX
```

Comment:
please re-generate from gitignore.io with the extra section headers

Reply:
Changes made